src/test/it/bom.yml

```diff
@@ -0,0 +1,56 @@
+metadata:
+  # labels and annotations are key: value string attributes
+  labels:
+    name: bom-it
+    # Needed to make PCT happy
+    groupId: org.jenkins-ci.main
+    artifactId: jenkins-war
+    version: 2.118-artifact-manager-s3-SNAPSHOT
+spec:
+  core:
+    version: 2.118
+  plugins:
+    - groupId: "org.jenkins-ci.plugins.workflow"
+      artifactId: "workflow-api"
+      version:  2.28-rc333.0675e9e9cb4c
```

Comment:
https://github.com/jenkinsci/jep/pull/92#discussion_r185943025 FTR:

```yaml
      branch: jglick:VirtualFile-JENKINS-49635
```

would be useful here in _addition_ to the `version`.